### DISCUSSION TITLE
2006/05/18/%E4%BD%BF%E7%94%A8-google-%E6%A1%8C%E9%9D%A2%E7%9A%84%E4%B8%80%E7%82%B9%E6%84%9F%E6%83%B3

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#84] [Announcements]
# 2006/05/18/%E4%BD%BF%E7%94%A8-google-%E6%A1%8C%E9%9D%A2%E7%9A%84%E4%B8%80%E7%82%B9%E6%84%9F%E6%83%B3

我们公司有很多软件和文件在公司的几台文件服务器上。经常有人需要安装一个软件时，不知道上哪去找。如果要根据文件内容找到一个以前放置在服务器上的文件就更困难了。

https://blog.qizhen.xyz/2006/05/18/%E4%BD%BF%E7%94%A8-google-%E6%A1%8C%E9%9D%A2%E7%9A%84%E4%B8%80%E7%82%B9%E6%84%9F%E6%83%B3

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Wu
2006年05月26日 09:37 编辑
挺赶时髦的嘛。我一直没有用过google desktop.
 
不过一直觉得企业内部的搜索工具是非常有必要的。以前在nish没有资源去做，如果现在的老板不把我炒掉的话，会在现在的公司试试。
 
到时交流心得，呵呵。
 
对了，google还有一个管理图片的工具吧？　用过吗？

回复
Qizhen
2006年05月24日 20:55 编辑
应该说是然 Google Desktop 共享给其他机器

回复
Qizhen
2006年05月24日 20:54 编辑
今天发现 有一个叫DNKA的插件可以让别人用浏览器访问其他机器上的 Google Desktop 。

回复
晓元
2006年05月20日 14:12 编辑
不需要每台机器都装吧，把索引放在一台机器上。
然后再写个jsp页面之类的，其他用户通过ie就可以来搜索，
intranet Google search
 
看完后发现，你已经有解决方法了:)
 
Enterprise search 里有一个有趣的领域叫专家搜索，比如当工作中遇到困难时，向谁寻求帮助是最合适的，在那些全球2，3十万人的公司里，这种expert search是很有用的。在nish里，找你就可以了吧，呵呵

回复
kl
2006年05月20日 14:04 编辑
好厉害！
